modules/docs/mdx/6.0-UPGRADE-GUIDE.mdx

```diff
@@ -76,29 +76,31 @@ help us maintain additional codemod transforms to make future upgrades easier.
 
 ### Deprecation Types
 
-There are two types of deprecations: soft and hard.
+There is `deprecated` and `removal`.
 
-#### Soft Deprecation
+#### Deprecation
 
-A soft-deprecated component is still available with its full functionality, but it will have been
-renamed with a prefix to indicate its soft-deprecated status. It will also include a console warning
-to announce its deprecation. This warning will only be triggered on the component's initial render.
+We add the [@deprecated](https://jsdoc.app/tags-deprecated.html) from JSDoc to code that we plan to
```

Comment:
Consider `@deprecated tag` for the link text.

Reply:
Updated